dkp-info.md

```diff
@@ -2,131 +2,141 @@
+has_toc: true
 ---
 
-# Industrial Peons DKP Priority System
+# DKP Info
 
-**\<Industrial Peons\>** uses a unique DKP auction system for 40 man raids. Our system discourages hoarding while ensuring no upgrade ever rots. In this guild, unlike most, we make the big decisions together democratically, so please read through this document and raise concerns you have. This document will be voted on, but it isn’t carved in stone, and if you can convince a majority of the guild, we can change any aspect of it (more on that in Section 3).
+**\<Industrial Peons\> uses a modified DKP auction system for 40 man raids, designed to discourage hoarding while ensuring no upgrade ever rots.**
 
-Our system was chosen after Peons voted to have a DKP system, but many also leaning towards a Loot Council system. So aspects of Loot Council have been incorporated into our DKP in the form of "priority classes/specs" for different items.
+Our system was chosen after Peons voted to have a DKP system, but with many also leaning towards a loot council system. To reconcile this, aspect of loot council have been incorporated into our DKP in the form of "priority classes/specs" for different items.
 
-**How DKP Is Awarded:** We will issue different DKP values per boss. For Phase 3, the following points will be awarded :
+## How DKP Is Awarded
 
-- 2 DKP points will be issued per each Molten Core boss kill.
+### Raids
 
-- 5 DKP points per each Onyxia kill.
+We issue different DKP values depending on whether the raid in question is _progression,_ _current,_ or _old._
 
-- 5 DKP for each BWL boss kill.
+- A raid is considered a _progression_ raid if it is the hardest available raid and we have yet to kill its final boss. Progression raids award **5 DKP per boss kill**, as well as **5 DKP per raiding hour** as hazard pay.
+- Once we kill the final boss of the most difficult currently available raid, that raid changes from _progression_ to _current._ Current raids still award **5 DKP per boss kill**, but no longer award the hazard pay.
+- Once a guild is no longer the hardest guild for the current phase, it becomes _old._ An old raid only awards **2 DKP per boss kill**, and old raids are periodically circulated out of our DKP system and replaced with GDKPs and soft res PUGs.
 
-**5 DKP will be awarded for being present in the raid before the first pull.** All new guild members will receive an initial seed of 60 DKP after they have completed their first raid with the Peons.
+We also have attendance modifiers for raids:
 
-**If you sign up for a raid, but fail to show without changing your sign-up to "absent", you will get -5 DKP for an “unexcused” absence.** If you cannot attend, please mark “absent” on the raid sign-up. It really helps us plan and makes everything go smoothly.
+- **5 DKP** will be awarded for being present in the raid before the first pull.
+- **60 DKP** will be given as a seed to new raid members at the end of their first raid in our DKP system.

```

Comment:
60 DKP will be given as a seed to any raiders who decide to join the Industrial Peons. Any new raiders who are not in the Peons will start at 0 DKP and accumulate from there